support-internationalisation/build.sbt

```diff
@@ -1,11 +1,16 @@
 import sbtrelease.ReleaseStateTransformations._
 
 name := "support-internationalisation"
+organization := "com.gu"
```

Comment:
this should have been set at the root level under ThisBuild so it should appear everywhere.